.storybook/main.js

```diff
@@ -1,4 +1,13 @@
+const IS_PRODUCTION = process.env.NODE_ENV === 'production'
+
+const stories = ["./**/*.stories.jsx"]
+
+// stories from src are for development purposes only
+if(!IS_PRODUCTION) {
```

Comment:
I added this check so that the deployed example doesn't have noise:
![image](https://user-images.githubusercontent.com/1671563/128092670-bb9a9b83-ec78-470d-a8fe-884e8176079d.png)